arch/risc-v/src/bl808/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+comment "BL808 Configuration Options"
+
+menu "BL808 Peripheral Support"
+
+config BL808_HAVE_UART3
+	bool "UART3"
+	select BL808_UART3
+	select ARCH_HAVE_UART3
+	select UART3_SERIALDRIVER
+	select ARCH_HAVE_SERIAL_TERMIOS
```

Comment:
@lupyuen since the chip has 3 UARTs it it important to include all UARTs here. Also the symbol BL808_HAVE_UART3 with this "_HAVE_" normally is used when we have a family of chips and some have and other don't have UART3. So it is better to name it as BL808_UART3. Please take a look at arch/xtensa/src/esp32/Kconfig as reference.

Reply:
Thanks @acassis I have changed BL808_HAVE_UART3 to BL808_UART3 and retested OK:

```text
config BL808_UART3
	bool "UART 3"
	default n
	select UART3_SERIALDRIVER
	select ARCH_HAVE_SERIAL_TERMIOS
```

As for the missing UART 0, 1, 2: Sorry the UARTs are little complicated on BL808. If we check the pic below, we see that NuttX runs on the 64-bit Core "D0", which is only connected to UART3. UART 0, 1 and 2 are not accessible because they are connected to the 32-bit Core "M0", which doesn't run NuttX.

I hope it's OK if we support only UART3 for NuttX on BL808? Thanks.

![BL808 RISC-V Cores](https://lupyuen.github.io/images/ox64-cores.jpg)